docs/docs/user-guides/whitelist-vpn.md

```diff
@@ -22,16 +22,21 @@ list grows.
 
 ## AWS Example
 
-In this example, a Firezone instance has already been set up on
-a `tc2.micro` EC2 instance.
-See the
-[Deployment Guide]({% link docs/deploy/index.md %})
-for details on deploying Firezone.
+Our goal is to configure VPN traffic to the restricted resource to be routed
+through a Firezone server on an EC2 instance.
```

Comment:
I think this sentence could be helpful in addition, which describes what role Firezone is playing in common lingo:

`In this case Firezone is acting as a network proxy or NAT gateway to provide a single public egress IP for all the devices connected to it.`